### PR TITLE
[libc++] Fix typo in _LIBCPP_REMOVE_TRANSITIVE_INCLUDES

### PR DESCRIPTION
**Pull request by bobsayshilol**
Spotted by inspection when trialling out `_LIBCPP_REMOVE_TRANSITIVE_INCLUDES`. No other instances of `_LIPCPP_` were found in the repo.

### CODE REVIEW

**Review by philnik777 — APPROVED** [review]

**Comment by bobsayshilol** [comment]
The failed job doesn't look relevant to my changes:
```
The runner has received a shutdown signal. This can happen when the runner service is stopped, or a manually started runner is canceled.
```
I don't see an option to retry though. Can anyone with perms give it a poke please?

**Comment by mordante** [comment]
> The failed job doesn't look relevant to my changes:
> 
> ```
> The runner has received a shutdown signal. This can happen when the runner service is stopped, or a manually started runner is canceled.
> ```
> 
> I don't see an option to retry though. Can anyone with perms give it a poke please?

Thanks for your contribution.

The CI sometimes has hiccups, but it's green. Do you need help merging the commit?

**Comment by bobsayshilol** [comment]
> Do you need help merging the commit?

Yes please. I don't have write access so I can't land it myself. Thanks!

**Pull request merged**
